.pylintrc

```
@@ -1,12 +1,83 @@
+[MASTER]
+
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code
+extension-pkg-whitelist=lxml,posix_ipc,spidev,netifaces
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+#ignore=
+
+# Add files or directories matching the regex patterns to the blacklist. The
+# regex matches against base names, not paths.
+#ignore-patterns=
+
+# List of plugins (as comma separated values of python module names) to load,
+# usually to register additional checkers.
+load-plugins=
+    pylint_strict_informational
+
+
 [MESSAGES CONTROL]
 
-disable=import-error,
-        missing-module-docstring,
-        missing-class-docstring,
-        missing-function-docstring,
-        no-name-in-module,
-        too-few-public-methods
+enable=
+    use-symbolic-message-instead,
+    useless-suppression
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line, not in the configuration
+# file where it should appear only once).You can also use "--disable=all" to
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+disable=
+    arguments-differ,
+    attribute-defined-outside-init,
+    bad-continuation,
+    cyclic-import,
+    duplicate-code,
+    fixme,
+    file-ignored,
+    import-outside-toplevel,
+    invalid-name,
+    locally-enabled,
+    locally-disabled,
+    missing-docstring,
+    no-init,
+    no-member,
+    no-self-use,
+    old-style-class,
+    protected-access,
+    redefined-variable-type,
+    superfluous-parens,
+    too-few-public-methods,
+    too-many-ancestors,
+    too-many-arguments,
+    too-many-branches,
+    too-many-instance-attributes,
+    too-many-lines,
+    too-many-locals,
+    too-many-public-methods,
+    too-many-statements,
+    ungrouped-imports,
+    unused-argument,
+    useless-return,
+    wrong-import-position,  # we have a custom isort config which pylint can't grok
+    wrong-import-order  # same reason
```

```suggestion
    wrong-import-order,  # same reason
    wrong-import-position  # we have a custom isort config which pylint can't grok
```

... pour ordonner alphabétiquement. ;)